Description: fix FTBFS related to -Wformat-security enabled by default
Author: Andrea Colangelo <warp10@ubuntu.com>

--- gbemol-0.3.2.orig/src/gbemol-utils.c
+++ gbemol-0.3.2/src/gbemol-utils.c
@@ -209,7 +209,7 @@ gboolean gbemol_cfg_save_file (GKeyFile*
 
 	if (cfg)
 	{
-		fprintf (cfg, data);
+		fprintf (cfg, "%s", data);
 		fclose (cfg);
 		g_free (path);
 		g_free (data);
@@ -447,7 +447,7 @@ GdkPixbuf* gbemol_utils_get_cover_image_
 		img = gdk_pixbuf_new_from_file (str, &e);
 		if (e) 
 		{
-			g_message (e->message);		
+			g_message (e->message, "%s");		
 			img = NULL;
 		}
 	}
@@ -479,7 +479,7 @@ GdkPixbuf* gbemol_utils_get_cover_image_
 		gbemol_net_download_file(result, str, &err);
 		if (err != NULL)
 		{
-			g_message(err->message);
+			g_message(err->message, "%s");
 		}
 		else
 		{
--- gbemol-0.3.2.orig/src/gbemol-app.c
+++ gbemol-0.3.2/src/gbemol-app.c
@@ -1134,7 +1134,7 @@ void gbemol_app_error_dialog_new (Gbemol
 	if (str1)
 		gtk_message_dialog_set_markup (GTK_MESSAGE_DIALOG (dlg), str1);
 	if (str2)
-		gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dlg), str2);
+		gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dlg), "%s", str2);
 
 	gtk_dialog_run (GTK_DIALOG (dlg));
 	gtk_widget_destroy (dlg);
@@ -1169,7 +1169,7 @@ void gbemol_app_reconnect_dialog_new (Gb
 	gtk_widget_show_all (btn);
 
 	gtk_message_dialog_set_markup (GTK_MESSAGE_DIALOG (dlg), str1);
-	gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dlg), str2);
+	gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dlg), "%s", str2);
 
 	if (gtk_dialog_run (GTK_DIALOG (dlg)) == GTK_RESPONSE_ACCEPT)
 		gbemol_app_connect_mpd (app);
